core/jobs/src/main/java/com/cognifide/aet/job/common/modifiers/executejavascript/ExternalSnippetHttpClient.java

```diff
@@ -37,42 +32,34 @@
 import org.apache.http.impl.client.CloseableHttpClient;
 import org.apache.http.impl.client.HttpClients;
 import org.apache.http.impl.conn.PoolingHttpClientConnectionManager;
-import org.apache.sling.commons.osgi.PropertiesUtil;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.metatype.annotations.Designate;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-@Component
-@Service(ExternalSnippetHttpClient.class)
+@Component(service = ExternalSnippetHttpClient.class)
+@Designate(ocd = ExternalSnippetHttpClientConf.class)
 public class ExternalSnippetHttpClient {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(ExternalSnippetHttpClient.class);
 
   private static final long DEFAULT_CONNECTION_TTL = 60L;
```

Comment:
Those values were moved to `ExternalSnippetHttpClientConf` so you probably can remove it from here.

Reply:
Removed